pkg/plugins/trivy/plugin_test.go

```diff
@@ -7697,6 +7717,12 @@ func TestGetInitContainers(t *testing.T) {
 				},
 			).Build()
 
+			// a number of init containers is 2 for Standalone, and 1 for ClientServer mode
+			numInitContainers := 2
+			if tc.configData["trivy.mode"] == string(trivy.ClientServer) {
+				numInitContainers = 1
+			}
+
```

Comment:
I think this test has way too many cases. Can we split it up based on client server vs standalone mode?

Reply:
i tried - https://github.com/aquasecurity/trivy-operator/pull/2595/commits/7cb23fbb4796540cad642f185ba687837e3042c6
but it looks not good for me (

Reply:
actually, it's splitted here now https://github.com/aquasecurity/trivy-operator/pull/2595/commits/7cb23fbb4796540cad642f185ba687837e3042c6